pyvista/core/filters/image_data.py

```diff
@@ -2848,7 +2848,7 @@ def _validate_dimensional_operation(
         # Build an array of the operation size
         operation_size = _validation.validate_array3(operation_size, reshape=True, broadcast=True)
 
-        if not isinstance(operation_mask, str):
+        if not isinstance(operation_mask, str) and operation_mask not in [0, 1, 2, 3]:
```

Comment:
This might come down to personal preference. If it doesn't resonate with you, feel free to disregard it.
```suggestion
        if not isinstance(operation_mask, str) and operation_mask not in range(4):
```

Reply:
This change is more compact but I have a slight preference for explicitly listing out the values in this case